homeassistant/components/device_tracker/device_action.py

```diff
@@ -0,0 +1,67 @@
+"""Provides device automations for Device tracker."""
+from typing import List, Optional
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_DEVICE_ID,
+    CONF_DOMAIN,
+    CONF_ENTITY_ID,
+    CONF_TYPE,
+    STATE_HOME,
+    STATE_NOT_HOME,
+)
+from homeassistant.core import Context, HomeAssistant, split_entity_id
+from homeassistant.helpers import entity_registry
+import homeassistant.helpers.config_validation as cv
+
+from . import DOMAIN, SERVICE_SEE
+from .const import ATTR_LOCATION_NAME, ATTR_DEV_ID
+
+ACTION_TYPES = {"set_home", "set_not_home"}
```

Comment:
Let's change these types to `comes_home` and `leaves_home`

Reply:
Good idea, will update all files accordingly. Do you know why the service `see` for `device_tracker` cannot be found? I am probably missing something trivial 😅

Reply:
Did you set up the device tracker integration?

Reply:
I am not quite sure what you mean by that. Only the second test `test_action` fails but I do not know why. Attempted to write the tests like this example https://github.com/home-assistant/home-assistant/pull/28550/files#diff-7e8a69c0a5ac211ea35c58f181662680
Where in `test_action` no entity has to be set up before mocking the service calls.